spec/std/isa/ext/A.yaml

```
@@ -128,3 +128,4 @@ params:
       the extension can be disabled in the `misa.A` bit.
     schema:
       type: boolean
+    extra_validation: assert MISA_CSR_IMPLEMENTED == true
```

MISA_CSR_IMPLEMENTED only needs to be true if MUTABLE_MISA_A is true:


```suggestion
    extra_validation: assert MISA_CSR_IMPLEMENTED == true if MUTABLE_MISA_A
```